packages/client/src/comp/states/BaseContainer.ts

```diff
@@ -80,11 +81,19 @@ const useFields = () => {
     canCheckRegistration() && fields.email.valid && fields.telephone.valid
   )
 
+  const name = (): Name => ({
```

Comment:
This doesn't need to be a function -- can just be a value

Reply:
I thought of doing this initially, but in the end we'd have to rework the type of `fields` (Right now it's a simple record of `<InputId, InputData>`, also we'd still end up needing a function to convert the stored `nameParts` with `inputData` inside `fields` to just a `NameParts`.

Reply:
I'm a little confused.  Why can't this be `nameParts: NameParts` and not a function?

Reply:
Sorry, I've overlooked a way to do this without changing the actual logic, submitting a commit to this right now.